src/librustdoc/lib.rs

```diff
@@ -261,6 +261,11 @@ pub fn opts() -> Vec<RustcOptGroup> {
                        "check if given theme is valid",
                        "FILES")
         }),
+        unstable("resource-suffix", |o| {
+            o.optopt("",
+                     "resource-suffix", "suffix which will be added at the end of resource files",
```

Comment:
This seems odd, both in verb tense and what we mean by "resource files". What about this?

> suffix to add to CSS and JavaScript files, e.g. "main.css" will become "main-suffix.css"

Reply:
It changed through the writing of this PR. Now we have an almost definitive version so it's more clear. I'll update to your sentence.